modules/cloudsql-instance/variables.tf

```diff
@@ -204,10 +210,15 @@ variable "network_config" {
   }
 }
 
-variable "postgres_client_certificates" {
-  description = "Map of cert keys connect to the application(s) using public IP."
-  type        = list(string)
+variable "ssl_mode" {
```

Comment:
Can we bundle this into the existing `network_config` variable? Or (maybe better), create a new top-level `ssl` variable and move certificates and SSL configuration there?

Reply:
I didnt like how `network_config` contained a lot of variables and validation would be complicated. I can move it into another ssl variable, as suggested, where I will put require_ssl, ssl_mode and certs, maybe that is a better approach

Reply:
Yep, I also agree this is the better option. If you could do it that would be great!